.travis.yml

```diff
@@ -19,10 +19,12 @@ matrix:
     - gemfile: gemfiles/Gemfile.public_suffix_2
       rvm: 2.5.5
   allow_failures:
+    - rvm: jruby-9.0.5.0
+    - rvm: jruby-9.1.17.0
```

Comment:
This version of jruby doesn't appear to be causing any issues anymore. The same applies to `9.2.6.0`.